py/torch_tensorrt/dynamo/debug/_Debugger.py

```diff
@@ -92,6 +98,26 @@ def __init__(
         self.capture_fx_graph_before = capture_fx_graph_before
         self.capture_fx_graph_after = capture_fx_graph_after
 
+        if self.cfg.capture_shim:
+            if not sys.platform.startswith("linux"):
+                _LOGGER.warning(
+                    "capture_shim featureis only supported on linux, will disable it"
+                )
+                self.cfg.capture_shim = False
+                return
+            if ENABLED_FEATURES.tensorrt_rtx:
+                _LOGGER.warning(
+                    "capture_shim feature is not supported on TensorRT-RTX, will disable it"
+                )
+                self.cfg.capture_shim = False
+                return
+            if not is_tensorrt_version_supported("10.13.0"):
+                _LOGGER.warning(
+                    "capture_shim feature is only supported on TensorRT 10.13 and above, will disable it"
```

Comment:
Capturing TensorRT API calls is only supported for TensorRT versions 10.13 and above, therefore ignoring the capture_tensorrt_api_recording setting